src/cloud-cli/meltano/cloud/cli/schedule.py

```diff
@@ -130,10 +203,174 @@ def enable(context: MeltanoCloudCLIContext) -> None:
 def disable(context: MeltanoCloudCLIContext) -> None:
     """Disable a Meltano Cloud schedule."""
     asyncio.run(
-        set_enabled_state(
+        _set_enabled_state(
             config=context.config,
             deployment_name=context.deployment,
             schedule_name=context.schedule,
             enabled=False,
         )
     )
+
+
+async def _get_schedule(
+    config: MeltanoCloudConfig,
+    deployment_name: str | None,
+    schedule_name: str | None,
+) -> CloudProjectSchedule:
+    if schedule_name is None:
+        raise click.UsageError("Missing option '--schedule'")
+    if deployment_name is None:
+        raise click.UsageError("Missing option '--deployment'")
+    async with SchedulesCloudClient(config=config) as client:
+        return await client.get_schedule(
+            deployment_name=deployment_name, schedule_name=schedule_name
+        )
+
+
+async def _get_schedules(
+    config: MeltanoCloudConfig,
+    deployment_name: str | None,
+    limit: int,
+) -> list[CloudProjectSchedule]:
+    page_token = None
+    page_size = min(limit, MAX_PAGE_SIZE)
+    results: list[CloudProjectSchedule] = []
+
+    async with SchedulesCloudClient(config=config) as client:
+        while True:
+            response = await client.get_schedules(
+                deployment_name=deployment_name,
+                page_size=page_size,
+                page_token=page_token,
+            )
+
+            results.extend(response["results"])
+
+            if response["pagination"] and len(results) < limit:
+                page_token = response["pagination"]["next_page_token"]
+            else:
+                break
+
+    return results[:limit]
+
+
+def _next_n_runs(n: int, cron_expr: str) -> tuple[datetime, ...]:
+    now = datetime.now(timezone.utc)
+    return tuple(it.islice(croniter(cron_expr, now, ret_type=datetime), n))
+
+
+def _approx_daily_freq(cron_expr: str) -> float:
+    now = datetime.now(timezone.utc)
+    num_days = 365
+    num_runs = sum(1 for _ in croniter_range(now, now + timedelta(num_days), cron_expr))
+    return num_runs / num_days
+
+
+def _process_table_row(schedule: CloudProjectSchedule) -> tuple[str | float, ...]:
+    return (
+        schedule["deployment_name"],
+        schedule["schedule_name"],
+        schedule["interval"],
+        _approx_daily_freq(schedule["interval"]),
+        schedule["enabled"],
+    )
+
+
+def _format_schedules_table(
+    schedules: list[CloudProjectSchedule], table_format: str
+) -> str:
+    """Format the schedules as a table.
+
+    Args:
+        schedules: The schedules to format.
+
+    Returns:
+        The formatted schedules.
+    """
+    return tabulate.tabulate(
+        [_process_table_row(schedule) for schedule in schedules],
+        headers=(
+            "Deployment",
+            "Schedule",
+            "Interval",
+            "Runs / Day",
+            "Enabled",
+        ),
+        tablefmt=table_format,
+        floatfmt=".3f",
```

Comment:
This level of precision is too high. Is it possible to either show no decimals when it's 0 or 1 when when it's not?

So `1.000` becomes `1` and `10.268` becomes `10.3`?

Reply:
That can be done, but I find the inconsistency in the output offputting. Here's what it looks like:

```
╭──────────────┬────────────┬─────────────────────┬──────────────┬───────────╮
│ Deployment   │ Schedule   │ Interval            │   Runs / Day │ Enabled   │
├──────────────┼────────────┼─────────────────────┼──────────────┼───────────┤
│ deployment 1 │ schedule 1 │ 1 2 * * *           │          1   │ True      │
│ deployment 2 │ schedule 2 │ 15,45 */2 * * 1,3,5 │         10.3 │ False     │
╰──────────────┴────────────┴─────────────────────┴──────────────┴───────────╯
```

Alternatively, we could round it all down to 1 decimal point of precision:

```
╭──────────────┬────────────┬─────────────────────┬──────────────┬───────────╮
│ Deployment   │ Schedule   │ Interval            │   Runs / Day │ Enabled   │
├──────────────┼────────────┼─────────────────────┼──────────────┼───────────┤
│ deployment 1 │ schedule 1 │ 1 2 * * *           │          1.0 │ True      │
│ deployment 2 │ schedule 2 │ 15,45 */2 * * 1,3,5 │         10.3 │ False     │
╰──────────────┴────────────┴─────────────────────┴──────────────┴───────────╯
```

Either way, I feel like this isn't enough precision. If you run a schedule weekly, for instance, it'll show `0.1`, which suggests that it's run once every 10 days, rather than once every 7. Running biweekly similarly shows `0.1`, and running monthly shows `0`.

I'd rather we not have a runs per day column at all than have the information be misleading.

Reply:
@WillDaSilva IMO the easiest resolution for intervals that don't run every day is to just do `< 1`. 

We could get really fancy with this and start to dive into significant figures, but that's overkill. For a pipeline that runs less than once / day we're not talking about a ton of money at all on Cloud, so the precision really isn't necessary.

Reply:
Updated in 9810f65